ngr_spider/ogc_api_features.py

```diff
@@ -27,7 +27,7 @@ def get_info(self):
         return Info(self.json["info"])
 
     def get_tags(self):
-        return self.json["tags"] | []
+        return self.json["tags"]
```

Comment:
Dit levert een KeyError op als tags niet bestaan. Ik denk dat het ok is om deze change te houden zoals het was. `self.json.get('tags', []) or []`